python/educational-core/student/src/com/jetbrains/edu/learning/courseGeneration/StudyProjectGenerator.java

```diff
@@ -126,7 +127,7 @@ public Course compute() throws RuntimeException {
         ProgressManager.getInstance().getProgressIndicator().setIndeterminate(true);
         return execCancelable(() -> {
 
-          final Course course = EduStepicConnector.getCourse(project, mySelectedCourseInfo);
+          final Course course = StepicConnectorGet.getCourse(project, mySelectedCourseInfo);
```

Comment:
Personally I do not like StepicConnectorGet name for the class because using its methods you always type "get" twice. Which is meaningless